.github/workflows/test.yml

```diff
@@ -144,7 +147,7 @@ jobs:
 
         run: cabal test wrapper-test --test-options="$TEST_OPTS --rerun-log-file .tasty-rerun-log-wrapper" || cabal test wrapper-test --test-options="$TEST_OPTS  --rerun-log-file .tasty-rerun-log-wrapper" || cabal test wrapper-test --test-options="$TEST_OPTS  --rerun-log-file .tasty-rerun-log-wrapper"
 
-      - if: matrix.test && matrix.ghc != '9.0.1'
+      - if: matrix.test && matrix.ghc != '9.0.1' && matrix.ghc != '9.0.2'
         name: Test hls-brittany-plugin
         run: cabal test hls-brittany-plugin --test-options="$TEST_OPTS" || cabal test hls-brittany-plugin --test-options="$TEST_OPTS" || LSP_TEST_LOG_COLOR=0 LSP_TEST_LOG_MESSAGES=true LSP_TEST_LOG_STDERR=true cabal test hls-brittany-plugin --test-options="$TEST_OPTS"
```

Comment:
Can we define a `cabal test` command that abstracts this stuff out. Ouch...

Reply:
maybe with a shell routine? or you are thinking in other possiblesolution?

Reply:
Oh hm, we would actually need a function here. I was thinking you could do something like
```
echo "CABAL_TEST=cabal test ..." >> $GITHUB_ENV
```
but I don't know if you can do something like that for bash functions :thinking:

Reply:
Ugly version would be to commit a small script that does that.

Reply:
There is nothing ugly in the ugly shell script in the `.workflow` directory.

I would post an argument that supplying a function through `env` or through `.bashrc` would be dirtier.

Also, the GHC is in a build environment, so it can be `ghci` script also.

It would be:
```shell
#!/usr/bin/env sh

BUILD_TARGET_NAME=$1

TEST_OPTIONS=$2

cabal test "$BUILD_TARGET_NAME" --test-options="$TEST_OPTIONS" || cabal test "$BUILD_TARGET_NAME" --test-options="$TEST_OPTIONS" || LSP_TEST_LOG_COLOR=0 LSP_TEST_LOG_MESSAGES=true LSP_TEST_LOG_STDERR=true cabal test "$BUILD_TARGET_NAME" --test-options="$TEST_OPTIONS"
```

```shell
[pyro@rogue haskell-language-server]$ bash -x ./draft.sh hls-brittany-plugin TOPT
+ BUILD_TARGET_NAME=hls-brittany-plugin
+ TEST_OPTIONS=TOPT
+ cabal test hls-brittany-plugin --test-options=TOPT
Resolving dependencies...
```

But I think this PR is already to big. Just an idea to refactor after merge.

Reply:
Reported in https://github.com/haskell/haskell-language-server/issues/2586